goth/runner/probe.py

```diff
@@ -192,14 +221,54 @@ async def create_app_key(self, key_name: str = "test_key") -> str:
             key = app_key.key
         return key
 
+    def set_agent_env_vars(self, env: Dict[str, str]) -> None:
+        """Add vars needed to talk to the daemon in this probe's container to `env`."""
+
+        if not self.app_key:
+            raise AttributeError("Yagna application key is not set yet")
+        path_var = env.get("PATH")
+        env.update(
+            {
+                "YAGNA_APPKEY": self.app_key,
+                "YAGNA_API_URL": f"http://{self.ip_address}:{YAGNA_REST_PORT}",
+                "GSB_URL": f"tcp://{self.ip_address}:{YAGNA_BUS_PORT}",
```

Comment:
You can use `YAGNA_REST_URL` and `YAGNA_BUS_URL` from `goth/address.py` here, like so:
```
"YAGNA_API_URL": YAGNA_REST_URL.substitute(host=self.ip_address)
```